app/frontend/src/features/profile/view/Overview.tsx

```diff
@@ -60,6 +61,13 @@ interface OverviewProps {
 
 export default function Overview({ user }: OverviewProps) {
   const classes = useStyles();
+  const [currentUser, setCurrentUser] = useState({} as User.AsObject);
+
+  useEffect(() => {
+    (async () => {
+      setCurrentUser(await getCurrentUser());
+    })();
+  }, []);
```

Comment:
```suggestion
  const currentUserId = useAuthContext().authState.userId;
```

My bad, I should have mentioned in the last review that this is available on the auth context already and as this is an authenticated page, it's safe to use/should always have a value.